test/spec.index.js

```diff
@@ -250,6 +250,57 @@ describe('Template Mixins', function () {
                 }));
             });
 
+            it('by default, assumes the field isn\'t required', function () {
```

Comment:
Cool. This would bring the the template-mixins in line with form-wizard. I don't foresee any implementation problems so I'll merge and publish as a major version.